plugin/plugins/mijia/mijia_api/infrastructure/cache_manager.py

```
@@ -0,0 +1,417 @@
+            for f in self._cache_dir.iterdir():
+                if f.is_file():
+                    try:
+                        import json as _json
+                        with open(f, "r", encoding="utf-8") as _f:
+                            data = _json.load(_f)
+                        if isinstance(data, dict) and "_value" in data:
+                            _ns = data.get("_namespace", "")
+                            if _ns == namespace or f.stem.startswith(namespace):
+                                f.unlink()
+                    except Exception:
+                        pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)